docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -1092,18 +1092,42 @@ When you are ready to add another application to the repo, you'll probably want
 
 You can also go through the full [Angular monorepo tutorial](/getting-started/tutorials/angular-monorepo-tutorial)
 
+## CI with Nx
```

Comment:
Should we call these differently, just also to optimize the search over Algolia?
Like depending on the tutorial name it "Setup CI for our Angular workspace" or similar?